lib/cisco_node_utils/interface_hsrp_group.rb

```diff
@@ -36,6 +36,7 @@ def initialize(interface, group_id, ip_type, instantiate=true)
 
```

Comment:
The following line of code needs to set instantiate to false.

https://github.com/cisco/cisco-network-node-utils/blob/3a8e8c5d4339e5faf5ab744cc413d23f3b59f3bc/lib/cisco_node_utils/interface_hsrp_group.rb#L48

`hash[intf][id][iptype] = InterfaceHsrpGroup.new(intf, id, iptype, false)`